pycds/__init__.py

```diff
@@ -121,7 +123,8 @@ class History(Base):
     observations = relationship(
         "Obs", backref=backref('meta_history', order_by=id))
 
-association_table = Table(
+# Association table for Obs *--* NativeFLag
+obs_raw_native_flags_t = Table(
```

Comment:
I wondered the same, but was feeling cautious. I'll search for other usages (expecting none) and use `ObsRawNativeFlags` throughout if none.